conda-forge.yml

```diff
@@ -1,14 +1,16 @@
 bot:
   automerge: true
 build_platform:
-  linux_64: linux_64
-  osx_64: osx_64
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
   osx_arm64: osx_64
-  win_64: win_64
```

Comment:
@jeongseok-meta, What is the reason to remove "win_64" here?

Reply:
I assumed that `X: X` is the default and only need to map overwrite for `X: Y`. If you prefer an explicit way, I can revert this change.

Reply:
You may be right. I am not sure how this part works. Does it still build for windows with this change?

Reply:
Yes. It does seem to do that. So I guess this is OK 👍

Reply:
I will let @pariterre merge this, as it is his library.

Reply:
Thanks a ton for contributing @jeongseok-meta

Reply:
Yes
- https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=970672&view=logs&j=171a126d-c574-5c8c-1269-ff3b989e923d
- https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=970672&view=logs&j=5be07ae1-d8ba-5406-47b6-8e3a3a12f825
- https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=970672&view=logs&j=e5cdccbf-4751-5a24-7406-185c9d30d021

`rerender` doesn't make any changes to the CI settings for `X: X`, so I presume removing it for them doesn't make any changes, including for linux64, osx64, and win64.

Reply:
> I assumed that X: X is the default and only need to map overwrite for X: Y. If you prefer an explicit way, I can revert this change.

I do prefer an explicit way :) 

Otherwise everything seems to be fine!

Reply:
Done!